backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/mobileui/picking/Pick_single_catch_weight_product.feature

```diff
@@ -53,8 +53,8 @@ Feature: mobileUI Picking - Pick catch weight products
       | PLV                    | product      | 5.0      | KGM               | CatchWeight          | Normal                        |
 
     And set mobile UI picking profile
-      | IsAllowPickingAnyHU | CreateShipmentPolicy  |
-      | Y                   | CREATE_COMPLETE_CLOSE |
+      | IsAllowPickingAnyHU | CreateShipmentPolicy  | IsAllowCompletingPartialPickingJob |
+      | Y                   | CREATE_COMPLETE_CLOSE | Y                                  |
```

Comment:
why changing the test input?
(assuming that by default IsAllowCompletingPartialPickingJob is considered N)

what about regression?

Reply:
(note: atm i am writting this, there is no other change in this file besides this one; no new test)